.github/workflows/native.yml

```
@@ -118,7 +118,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        rust: [1.74.0, stable]
+        rust: [1.77.2, stable]
```

```suggestion
        rust: [1.74.0, stable]
```
I would keep this one to 1.74.0 as it checks the compatibility with this version of the compiler